### DISCUSSION TITLE
Question about the content api

### DISCUSSION THREAD

**Discussion by ismaen** [GitbookIO/community#274] [0. General bugs and questions] [answered]
Can you programatically create a page and then insert content into that page using the gitbook API?

Essentially i'm trying to migrate my docs from another provider to gitbook, and i have both the Markdown and html, but can't seem to figure out the API to add to my script to automate the migration.

**Comment by addisonschultz** [comment, marked as answer]
If you're looking for something quick, you should be able to compress (zip) your markdown files, and import them using the import functionality! 

https://docs.gitbook.com/content-creation/import
